src/widgets/activity-group/model/hooks/useTimer.ts

```diff
@@ -0,0 +1,24 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { AppClockTimer } from '@app/shared/lib';
+
+const useTimer = () => {
+  const [, reRender] = useState(false);
+
+  const onTick = useCallback(() => {
+    reRender(x => !x);
+  }, []);
+
+  useEffect(() => {
+    const timer = new AppClockTimer(onTick, false);
+
+    timer.start();
+
+    return () => {
+      timer.stop();
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
You can remove the comment and move the `onTick` function inside the `useEffect` scope. No `eslint` warnings are expected in that case